tests/unit/lms/db/__init___test.py

```diff
@@ -43,12 +46,14 @@ def test_we_can_specify_keys_to_skip(self, model):
 
     def test_we_fail_to_update_when_skip_keys_is_not_a_set(self):
         with pytest.raises(TypeError):
-            ModelClass().update_from_dict({}, skip_keys=["a"])
+            ModelClass().update_from_dict(  # pylint:disable=no-member
+                {}, skip_keys=["a"]
+            )
 
     def test_repr(self):
         model = ModelClass(id=23, column=46)
 
-        assert repr(model) == ("ModelClass(" "id=23, " "column=46)")
```

Comment:
I'm not quite sure what's going on here. This was formatted crazily before, and now `make format` seems to have fixed it. Sometimes Black seems to be randomly kicking in and changing things that it wasn't before :man_shrugging: